config.schema.json

```diff
@@ -33,6 +33,24 @@
         "default": false,
         "required": true
       },
+      "pauseWord": {
+        "title": "Pause Word",
+        "type": "string",
+        "default": "pause",
+        "required": true
+      },
+      "findMe": {
+        "title": "Show switch for \"find me\"",
+        "type": "boolean",
+        "default": false,
+        "required": true
+      },
+      "findMeWord": {
+        "title": "Find Me Word",
+        "type": "string",
+        "default": "where are you",
+        "required": true
+      },
```

Comment:
Do you mind adding a bit of documentation about these new options in the README?

Reply:
Like the idea ;-) Done !